tests/dragonfly/requirements.txt

```diff
@@ -19,7 +19,7 @@ numpy==1.24.3
 pytest-json-report==1.5.0
 psutil==5.9.5
 boto3==1.28.55
-redis-om==0.2.1
+redis-om==0.2.2
```

Comment:
This is needed -- otherwise `requirements` can't be installed since redis cluent 5.0 requires `0.2.2`